.gitlab/ornl-ci.yml

```diff
@@ -1,93 +1,146 @@
 
 stages:
   - setup
+  - compile
   - update
-  - build ascent
+  - build
   - test
-  - clean
+  - cleanup
 
-Spack Setup:
+.final_steps:
+  script: &finalize
+    - chgrp -fR exaworks $SPACK_WORK_DIR/spack || true
```

Comment:
group should be `csc449` (project for exaworks)

Reply:
Thanks, fixed.